tests/components/influxdb/test_init.py

```diff
@@ -21,6 +23,7 @@
     "organization": "org",
     "token": "token",
 }
+FilterTest = namedtuple("FilterTest", "id should_pass")
```

Comment:
We can use a dataclasses class instead of a named tuple.

https://docs.python.org/3/library/dataclasses.html

Reply:
Oh that's neat, I didn't know about that.

In https://github.com/home-assistant/core/pull/36913 I just merged a bunch of tests just like these where I used a `namedtuple` called `FilterTest` in exactly the same way. Seems like conversion from one to the other is trivial. Can I fix those as well?  It would add 3 lines of change in the following four files:
- `tests/components/apache_kafka/test_init.py`
- `tests/components/azure_event_hub/test_init.py`
- `tests/components/google_pubsub/test_init.py`
- `tests/components/prometheus/test_init.py`

Reply:
Please make a separate PR for that clean up. Thanks!